steps/genymotion-cloud-saas-stop/0.0.1/step.yml

```diff
@@ -0,0 +1,38 @@
+title: Genymotion Cloud Stop Android Devices
+summary: |
+  Stop Genymotion Cloud SaaS instances
+description: |
+  Stop Genymotion Cloud SaaS instances.
+website: https://github.com/genymobile/bitrise-step-genymotion-cloud-saas-stop
+source_code_url: https://github.com/genymobile/bitrise-step-genymotion-cloud-saas-stop
+support_url: https://github.com/genymobile/bitrise-step-genymotion-cloud-saas-stop/issues
+published_at: 2020-01-06T17:36:37.864519054+01:00
+source:
+  git: https://github.com/genymobile/bitrise-step-genymotion-cloud-saas-stop.git
+  commit: 4e7246a95229ee45d404abe73e6d7e76f8b128d6
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- android
+- xamarin
+- react-native
+- cordova
+- ionic
+- flutter
+type_tags:
+- utility
```

Comment:
It should be be probably `test`

Reply:
see: https://github.com/bitrise-io/bitrise-steplib/pull/2344#discussion_r363683035

Reply:
see: https://github.com/bitrise-io/bitrise-steplib/pull/1101#issuecomment-314151310